src/components/shared/image.js

```diff
@@ -76,20 +91,34 @@ class OImage extends Component {
           }
         })
         .catch(error => console.log("error in OImage:isExist"));
-    } else this.setState({ source: require("../../images/no-image-featured-image.png") });
+    } else this.setState({ source: placeholderImage });
   }
 
   displayImage() {
+    const { source, loading } = this.state;
+    const { style } = this.props;
+
+    let width = 0;
+    let height = 0;
+
```

Comment:
"style" was never really used here, so it didn't matter that it was formatted in different ways depending on where it came from. If there's only one day left of bugfixes, then I'm afraid there won't be time to make the whole image loading process less clunky.